web/js/codeworld_shared.js

```diff
@@ -1210,31 +1247,43 @@ function createFolder(path, buildMode, successFunc) {
             sweetAlert.close();
             const data = new FormData();
             data.append('mode', buildMode);
-            if (path === '') {
-                data.append('path', result.value);
-            } else {
+
+            path === '' ?
```

Comment:
I don't think you should use the ternary operator when you're ignoring the expression's value.  if/else is the right tool here.